### ISSUE TITLE
accessibility - navbar

### ISSUE THREAD

**Issue by leonawicz** [ropensci-org/rotemplate#34]
I saw issue #14 but I think there should be more consideration of accessibility. I wanted to submit a new issue because the low navbar contrast actually makes it very difficult to read without hovering over each item (which turns them white- they should probably be white already).

I am not an expert on accessibility either by any means, but there are great [tools](https://wave.webaim.org/report#/https://docs.ropensci.org/eia/) that help a significantly with getting web content closer to [WCAG standards](https://www.w3.org/WAI/standards-guidelines/wcag/)

I understand if we do not have the resources or expertise to make our pages meet every single criterion. None of the web content I produce does either. But I feel like the low contrast navbar is pretty egregious currently and as a low-hanging fruit we could easily do better there.

![low_contrast](https://user-images.githubusercontent.com/1672103/70474693-514f8480-1a90-11ea-8c8e-c03e1fddd8b4.png)

Making the font white and even bold would make a huge difference. Technically, even that will still fail the contrast test, due to the lightness of the blue background. But still a major improvement. Hovering over items can always be indicated by changing the background color rather than the foreground text, similar to how selected items change. Again, I am not an expert either, so there is nothing special about these particular suggestions.

Anyway, it does sound like people have already met to discuss accessibility, but while there are seemingly countless accessibility guideline items, some could be prioritized to address and are very easy to do so. This one seems very pertinent. While I do have outlier-level poor vision, I at least benefit from corrective lenses (though they do make everything I see appear notably smaller in size than what most people see). There will be plenty of people who find the navbar more difficult to read than I do. I just would like to encourage revisiting this and adoption of some of these tools if they have not been used previously.

**Comment by jeroen** [comment]
The latest docs use a darker color navbar, e.g: https://docs.ropensci.org/magick. I also triggered a docs build for eia (you might need to ctrl+shift+R to force a refresh in the browser to get the new colors).

However the text is still a light shade of blue, not white. These are colours that our designer came up with, I guess we can change it to white if that helps.

We have indeed discussed accessibility in our team, but concluded that we really don't know anything about this and don't want to start randomly start stabbing in our design based on guess work without consulting an expert.

Also I was under the impression that users with e.g. low color vision would probably use screen settings or browser plugins to increase the contrast or default text size so there wouldn't be much point in increasing the contrast on our side. Is this not the case?

**Comment by maelle** [comment]
It can't hurt if we make the text in the navbar white, and the rectangle around text lighter on hover As @leonawicz suggests? Also quite quick to do? But will it be enough contrast even then?

@leonawicz we are also working on a redesign of the main website, whose new design will then inform changes in this template.

**Comment by leonawicz** [comment]
@maelle I just checked. Technically, using the new darker blue background and white text, it only meets the minimum contrast standard for large, bold font.

@maelle @jeroen Since it's in the middle of a broader website redesign, it seems there is no better time than now to consider accessibility accommodations. I guess we can close this issue if it is more pertinent to another repo for the overall ROpenSci website. But would it be okay if I opened a discussion in one of our Slack channels to share my thoughts on accessibility accommodations? Like I said, I'm no expert on all the criteria, but I work with people who are daily and have been in many accessibility discussions. If so, is there a certain channel that would be best for such a conversation?

**Comment by maelle** [comment]
@leonawicz I'm sorry to have left you hanging!! I totally messed up.

We have a new website nearly ready for launch with better contrast in the navbar (AA only though :-( ) https://roweb3-hugo.netlify.app

And I made a PR to use similar colors for rotemplate #43

**Comment by maelle** [comment]
One thing we also recently tweaked for contrast is the code highlighting style of the main website btw.
